osbs/utils.py

```diff
@@ -142,6 +142,18 @@ def graceful_chain_del(d, *args):
         pass
 
 
+def has_triggers(x):
+    if x.get('spec', {}).get('triggers'):
+        return True
+    return False
```

Comment:
Could use `return graceful_chain_get(x, 'spec', 'triggers') is not None`. But also, can we call it `build_config` instead of `x`?